src/test/java/org/jabref/model/entry/CrossrefTest.java

```diff
@@ -0,0 +1,85 @@
+package org.jabref.model.entry;
+
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.jabref.logic.importer.ImportException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.logic.importer.ImportFormatReader;
+import org.jabref.logic.xmp.XmpPreferences;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.util.DummyFileUpdateMonitor;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockito.Answers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class CrossrefTest {
+
+    private static BibDatabase database;
+
+    @BeforeAll
+    static void SetUp() throws ImportException, URISyntaxException {
+        ImportFormatReader reader = new ImportFormatReader();
+        ImportFormatPreferences importFormatPreferences = mock(ImportFormatPreferences.class, Answers.RETURNS_DEEP_STUBS);
+        when(importFormatPreferences.getEncoding()).thenReturn(StandardCharsets.UTF_8);
+        reader.resetImportFormats(importFormatPreferences, mock(XmpPreferences.class), new DummyFileUpdateMonitor());
+
+        Path file = Paths.get(CrossrefTest.class.getResource("crossref.bib").toURI());
+        database = reader.importFromFile("bibtex", file).getDatabase();
```

Comment:
Please create the database by hand (i.e. `new BibEntry(...)`) instead of parsing a bib-file. This has also the advantage that you can keep a reference of the entries and don't need the `getEntry` method.